### ISSUE TITLE
com.github.scredis#scredis_2.12;2.1.1: not found

### ISSUE THREAD

**Issue by almoehi** [scredis/scredis#31]
Hi,
according to the docs scredis v2.1.1 should be scala 2.12 compatible.
However, when using the dependency as shown on the docs page it can't resolve it.

I manually checked sonatype, it seems like there are no published artifacts for 2.1.1 - am I missing something ?

```
scalaVersion := "2.12.3"

libraryDependencies ++= Seq(
  ws,
  guice,
  "com.typesafe.play" %% "play-json" % "2.6.5",
  "com.typesafe.play" %% "play-iteratees" % "2.6.1",
  "com.github.scredis" %% "scredis" % "2.1.1"
)

resolvers ++= Seq(
  Resolver.sonatypeRepo("public"),
  Resolver.bintrayRepo("scalaz", "releases"),
  Resolver.bintrayRepo("jastice", "releases")
)
```

**Comment by jastice** [comment]
yeah I still haven't published it to sonatype... but doesn't it work via the bintray resolver?

**Comment by almoehi** [comment]
Nope - I ended up to do a publishLocal from the repo

**Comment by mazorigal** [comment]
Hi, 
any update about the availability of the new version ?

**Comment by jastice** [comment]
I'm happy to make anyone a maintainer willing to take care of it :)

**Comment by faolivera** [comment]
Hi @jastice I would like to take care of this.

Best,
Facundo

**Comment by jastice** [comment]
@faolivera Great! I've added you as collaborator

**Comment by eugenemiretsky** [comment]
@faolivera, are you still maintaining this? 
Looking for the new version as well

**Comment by jastice** [comment]
@eugenemiretsky would you like to take care of it? :)
